tests/smoke/react-v14/index.js

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env node
+
+const React = require('react');
+const reactElementToJsxString = require('./../../../dist/').default;
```

Comment:
It's ugly but at the same time you are testing the build! Which is good.

Reply:
Do we need ".default"? I did not remember that, maybe because I was always using it from a babel transpiled frontend app.

Might be good to be able to require it without default in commonJS

Reply:
The `default` is mandatory. I have no idea if this possible to avoid using it. I will create a dedicated issue for this

Reply:
See https://github.com/algolia/react-element-to-jsx-string/issues/271